app/views/games/dashboard.html.erb

```diff
@@ -1,111 +1,90 @@
+    <div class="terror-border">
+      <div class="terror-bar" ng-class="getTerrorColour()" ng-style="getTerrorWidth()"></div>
+    </div>
+  </div>
+</div>
 
-        <!-- This needs to be formatted for use -->
+  <!-- Control Messages -->
+  <div id="control-messages">
+    <p>{{controlMessage}}</p>
+  </div>
 
-        <h3>Turn {{round}}</h3>
-        <h3 class="paused" ng-if="paused">(Paused)</h3>
-      </div>
 
-      <h3>News:</h3>
+  <!-- News -->
+  <div id="news-block">
 
-      <div class="news-container" ng-show="news.length>0" ng-repeat="news_item in news" ng-class="{ active: $index == 0 }">
-        <h4>{{news_item.title}}</h4>
-        <img ng-show="{{news_item.media_url.length>0}}" ng-src="{{news_item.media_url}}" />
-        <p>{{news_item.content}}</p>
-        <p class="date">{{news_item.created_at| date:'h:mma'}}</p>
-      </div>
+    <div id="breaking-block">
+      <h3>Breaking News</h3>
+      <div class="triangle"></div>
+    </div>
 
-      <div class="news-container active" ng-if="news.length==0">
-        <h4>Nothing has happened yet<h4>
+    <div id="news-image" ng-class="news[newsIndex].media_landscape == true ? 'landscape' : 'portrait'" ng-if="news[newsIndex].media_url">
+      <div class="image"
+           ng-show="news[newsIndex].media_url"
+           style="background-image: url({{news[newsIndex].media_url}})">
       </div>
-      <!-- End News -->
     </div>
 
-    <div class="right-column">
-      <div id="riot-top"></div>
-      <img id="riot-image" src="/assets/riot.png" />
-
-      <div id="rioters-container">
-        <h3>Rioters</h3>
-        <p>{{rioters}}</p>
-
-        <div id="rioters-count">
-          <span ng-repeat="a in range(rioters) track by $index">
-            <img ng-if="$index<10" src="/assets/riotcount.png" />
-          </span>
-          <span ng-if="rioters>10" class="additional-rioters">+</span>
-        </div>
-
-        <div id="activity-container">
-        <img id="activity-image" src="/assets/activity.png" />
-           <h3>Military Activity</h3>
-          <p>{{activity}}</p>
+    <div id="news-container">
+      <div class="slider" ng-if="news.length>0" ng-class="news[newsIndex].media_landscape == false && news[newsIndex].media_url ? 'portrait' : 'landscape'">
+        <div ng-repeat="news_item in news"
+             class="slide slide-animation nonDraggableImage"
+             ng-hide="!isCurrentSlideIndex($index)">
+          <h4>{{news_item.title}}</h4>
+          <p>{{news_item.content}}</p>
+          <p class="news-time">Reported at {{news[newsIndex].created_at| date:'h:mma'}}</p>
         </div>
       </div>
+      <div class="" ng-if="news.length==0">
+        <h4>Reports are coming in. Stay tuned for more details.</h4>
+      </div>
     </div>
   </div>
-</div>
+
+  <!-- Countries Ticker -->
+  <div id="countries-ticker">
+    <ul>
+      <li>
+        <div ng-repeat="country in countries" class="country">
+          <span>{{country.name}}</span>
```

Comment:
I'm not sure if I like Country Name, Amount, then Glyphicon.  I think maybe a | as a delimiter would be a good idea between countries.